pkg/sqlite/scene.go

```diff
@@ -1369,15 +1369,15 @@ func scenePerformerTagsCriterionHandler(qb *SceneStore, tags *models.Hierarchica
 
 			valuesClause := getHierarchicalValues(ctx, qb.tx, tags.Value, tagTable, "tags_relations", "", tags.Depth)
 
-			f.addWith(`performer_tags AS (
+			f.addWith(`performers_tags AS (
```

Comment:
I'm by no means a SQL expert so I could be 1000% wrong on all of this but:

Maybe I'm misunderstanding the original intent in this query, but isn't `performer_tags` here an alias of the query in parens as opposed to a reference to the table? Note that the reference to the table on line 1374 is to the correct table name. Now once we get to the bottom two changes in this file I'd expect/assume the `addLeftJoin` change is correct (unless you can join on an aliased sub-query?) and the `addHierarchicalConditionClauses` I'd have to coin flip without immersing myself in this codebase.